packages/components/src/components/hds/table/th-selectable.hbs

```diff
@@ -4,13 +4,18 @@
 }}
 
 <Hds::Table::Th class="hds-table__th--is-selectable" @scope={{@selectionScope}} ...attributes>
```

Comment:
Issue: the th is missing aria-sort='ascending/descending' if the column is sorted by the selected items. you can see it for the other sortable columns here: https://github.com/hashicorp/design-system/blob/a5e990eda421353d1f20f9c546c99fa8e5547780/packages/components/src/components/hds/table/th-sort.hbs#L7